lib-find-meal/build.gradle.kts

```diff
@@ -0,0 +1,53 @@
+plugins {
```

Comment:
There's a lot of repeated stuff in these gradle files, I can create a gradle file that has all the common stuff like compileSdkVersion etc so we can just include that file in each module

Reply:
Struggling to get this working nicely with the .kts stuff

Reply:
When we did the tutorial to set up a skeleton project we had to refer to the many gradle files for different plugins/dependencies so I'm not sure how simple it would be to combine?

Reply:
In a non-kts gradle file you can just apply from another .gradle file so it's easy to just put all the common dependencies etc in one file and just apply it, but I can't seem to get it to work with kts. I guess for now we can just repeat ourselves but would be nice to fix!

Reply:
The blueprint does it (not with .kts extensions though) https://bitbucket.org/theappbusiness/android-blueprint/src/develop/buildScripts/common.gradle